### ISSUE TITLE
Add number of remaining features for feature importances

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#109]
Just like we have it for feature weights, this could show the number of remaining features with non-zero feature importance - it can still be lower than total number of features, I think.

**Comment by kmike** [comment]
It makes sense to me. 
Do you think we should show an overall number of features, or a number of non-zero features?

**Comment by lopuhin** [comment]
I was thinking of showing the number of non-zero features that are not listed explicitly. The overall number of features just comes from the vectorizer or input, or that is something different in context of trees?

**Comment by kmike** [comment]
No, I don't think it is any different. +1 to show a number of remaining non-zero features.
